docs/tutorial.md

```diff
@@ -649,47 +688,6 @@ the cluster one at at time.  Others may permit parallel deployment.
 The DC/OS Stateful Services SDK includes deployment strategies supporting
 these different requirements.
 
-## Configuration management
```

Comment:
"Configuration management" is still a feature, and we should include it here.  But only at a high level.  Only the "how to" part should be moved.